lib/Config.js

```diff
@@ -82,6 +82,9 @@ class Config extends EventEmitter {
             this.bootstrapList = [];
         }
 
+        if (process.env.SITE_NAME) {
```

Comment:
I think the environment variable should better explain what it does exactly (if I understand correctly, we want to restrict the list of locations to be managed, so we could have something like `MANAGED_SITE_NAME` (but please challenge that)